customers/models.py

```diff
@@ -258,6 +258,12 @@ class Meta:
         verbose_name_plural = _("customer profiles")
         ordering = ("id",)
 
+    def is_non_billable_customer(self):
```

Comment:
Would it make sense to turn this into a `@prop` instead?

Reply:
I think there is no benefit because we are not exposing the field. what do you think / what would be the benefit?

Reply:
🤔 Good point. In my mind, it is a property of the customer, meaning that the customer `either is a billable customer or not`, but there's no real benefit/downside of keeping it as either,